deployment/ansible/group_vars/all

```diff
@@ -18,6 +18,9 @@ postgresql_host: "database.service.driver.internal"
 postgresql_port: 5432
 postgresql_version: "9.4"
 
+redis_bind_address: "database.service.driver.internal"
```

Comment:
It's not a big deal since this will likely get overridden in most cases anyway, but it'd probably be cleaner to have this be `cache.service...` or `redis.service...` since there's no guarantee the redis host will be the same as the database host.